tests/post/features/backup.feature

```diff
@@ -0,0 +1,5 @@
+@post @ci @local @slow @backup
+Feature Backup
+    Scenario: Backup multiple times
+        Given we run the backup script 6 times
+        Then we have 5 backups
```

Comment:
Ideally, we should wait here for the replication job to complete and check for number of backups on every nodes